2021-02-04 17:09:18.659657: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
--- Logging error ---
Traceback (most recent call last):
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/scratch/brussel/102/vsc10255/Experimental-Reactivity-Prediction/code/models/CM_ZPE_3DCNN.py", line 109, in <module>
    log.info('Platform: ', platform.platform())
Message: 'Platform: '
Arguments: ('Linux-3.10.0-1160.2.2.el7.x86_64-x86_64-with-glibc2.2.5',)
--- Logging error ---
Traceback (most recent call last):
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/scratch/brussel/102/vsc10255/Experimental-Reactivity-Prediction/code/models/CM_ZPE_3DCNN.py", line 110, in <module>
    log.info('OS: ', platform.system())
Message: 'OS: '
Arguments: ('Linux',)
--- Logging error ---
Traceback (most recent call last):
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/scratch/brussel/102/vsc10255/Experimental-Reactivity-Prediction/code/models/CM_ZPE_3DCNN.py", line 111, in <module>
    log.info('Version: ', platform.version())
Message: 'Version: '
Arguments: ('#1 SMP Tue Oct 20 16:53:08 UTC 2020',)
--- Logging error ---
Traceback (most recent call last):
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
  File "/apps/brussel/CO7/broadwell/software/Python/3.8.2-GCCcore-9.3.0/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/scratch/brussel/102/vsc10255/Experimental-Reactivity-Prediction/code/models/CM_ZPE_3DCNN.py", line 112, in <module>
    log.info('Processor: ', platform.processor())
Message: 'Processor: '
Arguments: ('x86_64',)
/user/brussel/102/vsc10255/.local/lib/python3.8/site-packages/qubit/preprocessing/matrix_operations.py:32: Warning: Trying to reduce the matrix size, default matrix size has been returned!
  warnings.warn(
WARNING:tensorflow:From /apps/brussel/CO7/broadwell/software/TensorFlow/2.3.1-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2021-02-05 04:53:28.632325: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /apps/brussel/CO7/broadwell/software/TensorFlow/2.3.1-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
INFO:tensorflow:Assets written to: /scratch/brussel/102/vsc10255/Experimental-Reactivity-Prediction/code/models/../../models/CM_ZPE_3DCNN/3D_CNN_0.1.3_on_02-04-2021_17.03.51.tf/assets
